feature/my/src/main/java/com/hankki/feature/my/component/JogboFolder.kt

```diff
@@ -71,11 +70,8 @@ fun JogboFolder(
                 modifier = Modifier.padding(start = 20.dp),
                 horizontalArrangement = Arrangement.spacedBy(6.dp)
             ) {
-                item {
-                    JogboHashtagChip(chiptext = chip1)
-                }
-                item {
-                    JogboHashtagChip(chiptext = chip2)
+                items(chips.size) { index ->
+                    JogboHashtagChip(chiptext = chips[index])
```

Comment:
```suggestion
                items(chips) { chip ->
                    JogboHashtagChip(chiptext = chip)
```
items에 list를 넣어줄 수 있어요